src/include/duckdb/execution/operator/schema/physical_create_index.hpp

```diff
@@ -34,13 +34,13 @@ class PhysicalCreateIndex : public PhysicalOperator {
 	//! The list of column IDs required for the index
 	vector<column_t> column_ids;
 	//! Set of expressions to index by
-	vector<unique_ptr<Expression>> expressions;
+	mutable vector<unique_ptr<Expression>> expressions;
```

Comment:
Yes, because it has to be passed to the ART index constructor, which in turn uses an `ExpressionEvaluator`, which can apparently mutate the expressions (if it doesn't that will be my next trick!)

Reply:
That is perhaps a future target for `const`ness, the expression executor shouldn't actually mutate expressions themselves!